sec: -49292
x_max: 3.19353
x_min: -3.85558
y_max: 3.20182
y_min: -3.67197
z_max: 1.10198
z_min: -5.74987
